src/components/Teaser/__tests__/Teaser.spec.tsx

```diff
@@ -0,0 +1,153 @@
+describe('Teaser', () => {
+    it('should render correctly with just a title', () => {
+        const { container } = render(<Teaser title="some title" />);
+
+        expect(container).toMatchSnapshot();
+    });
+
+    it('should render with all props defined', () => {
+        const { container } = render(
+            <Teaser
+                title="A job title"
+                location="location"
+                subTitle="Organization"
+                details="details about this job"
+                statuses={[
+                    {
+                        label: 'Viewed',
+                        tooltip: 'Viewed two days ago',
+                    },
+                    {
+                        label: 'Imported',
+                        tooltip: 'Imported five days ago',
+                    },
+                ]}
+            />
+        );
+
+        expect(container).toMatchSnapshot();
+
+        const title = screen.getByTitle('A job title');
+        const location = screen.getByText('location');
+        const subTitle = screen.getByText('Organization');
+        const details = screen.getByText('details about this job');
+
+        expect(title).toBeInTheDocument();
+        expect(title).toHaveClass('Teaser__title');
+        expect(title).not.toHaveClass('Text--context_neutral');
+
+        expect(location).toBeInTheDocument();
+        expect(location).toHaveClass('Text--context_neutral', 'Teaser__location');
+
+        expect(subTitle).toBeInTheDocument();
+        expect(subTitle).toHaveClass('Teaser__subTitle');
+        expect(subTitle).not.toHaveClass('Text--context_neutral');
+
+        expect(details).toBeInTheDocument();
+        expect(details).toHaveClass('Text--context_neutral', 'Teaser__details');
+    });
+
+    it('should render correctly in disabled mode', () => {
+        const { container } = render(
+            <Teaser
+                title="A job title"
+                location="location"
+                subTitle="Organization"
+                details="details about this job"
+                disabled
+            />
+        );
+
+        expect(container).toMatchSnapshot();
+
+        const title = screen.getByTitle('A job title');
+        const location = screen.getByText('location');
+        const subTitle = screen.getByText('Organization');
+        const details = screen.getByText('details about this job');
+
+        expect(title).toBeInTheDocument();
+        expect(title).toHaveClass('Text--context_neutral', 'Teaser__title');
+
+        expect(location).toBeInTheDocument();
+        expect(location).toHaveClass('Text--context_neutral', 'Teaser__location');
+
+        expect(subTitle).toBeInTheDocument();
+        expect(subTitle).toHaveClass('Teaser__subTitle', 'Text--context_neutral');
+
+        expect(details).toBeInTheDocument();
+        expect(details).toHaveClass('Text--context_neutral', 'Teaser__details');
+    });
+
+    it('should render correctly in disabled mode', () => {
```

Comment:
duplicate test? I think it is the same as the previous one :-P

Reply:
It was just for redundancy in case the first failed 😄